### PR TITLE
Check if the server has user status app enabled

### PR DESCRIPTION
**Pull request by FlexW**
Signed-off-by: Felix Weilbach <felix.weilbach@nextcloud.com>

### CODE REVIEW

File: src/gui/tray/usermodel.cpp [hunk 1]
```diff
@@ -658,7 +658,7 @@ QString User::statusEmoji() const
 
 bool User::serverHasUserStatus() const
 {
-    return _account->account()->capabilities().userStatusNotification();
+    return _account->account()->capabilities().userStatus();
```
Comment (mgallien) [inline src/gui/tray/usermodel.cpp:661]: should we not use:
```
return _account->account()->capabilities().userStatus() && account->account()->capabilities().userStatusNotification();
```

Comment (FlexW) [reply src/gui/tray/usermodel.cpp:661]: Why? The docs just say we should check the capabilities. We don't check the endpoints for other capabilities.

Comment (mgallien) [reply src/gui/tray/usermodel.cpp:661]: OK

**Review by mgallien — APPROVED** [review]

**Review by allexzander — APPROVED** [review]

**Pull request merged**